components/icons/sidebar_icons.tsx

```diff
@@ -84,3 +84,20 @@ export const ExitIcon: React.FC = ({}) => (
     />
   </svg>
 );
+
+export const LeftChevron: React.FC = ({}) => (
+  <svg
+    width="10"
+    height="28"
+    viewBox="0 0 10 16"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
My bad because I didn't catch this on first review. React wants these to be camel case.
```suggestion
      fillRule="evenodd"
      clipRule="evenodd"
```